rpcs3/Emu/NP/rpcn_config.cpp

```diff
@@ -106,6 +106,255 @@ std::vector<std::pair<std::string, std::string>> cfg_rpcn::get_hosts()
 	return vec_hosts;
 }
 
+std::string cfg_rpcn::get_country() const
+{
+	return country.to_string();
+}
+
+std::vector<cfg_rpcn::country_code> cfg_rpcn::get_countries()
+{
+	std::vector<country_code> countries;
+	countries.push_back({"United States", {'u','s'}});
```

Comment:
I don't understand why you don't simply pass "us" though, if this is supposed to be a std::string
```suggestion
	countries.push_back({"United States", {'u', 's'}});
```

Reply:
This was a relic of trying to populate the code in a s32. I switched back to string as Qt didn't like when i passed it that way and seemingly had no conversion.